pyproject.toml

```diff
@@ -32,6 +32,7 @@ pyicu = [
 ]
 natsort = "^8.4.0"
 univers = "30.12.1"
+cyclonedx-python-lib = "7.6.0"
```

Comment:
Please bump the version to the most current one `8.3.0`, just to make sure that we won't face problems after dependabot increases it.

Reply:
Done